### ISSUE TITLE
Feature request: autocomplete placeholder selectors

### ISSUE THREAD

**Issue by tarciozemel** [mrmlnc/vscode-scss#52]
Would be nice if autocomplete also triggered in placeholder selectors.

**Comment by zmanring** [comment]
Any movement on this?

**Comment by dmstern** [comment]
And not just autocomplete, also the ctrl+click on something like `%my-extend` should work.

**Comment by rans0r** [comment]
After looking into this for a while myself, I'm not sure this is possible. Since placeholder selectors can be dynamically named using variables the SCSS compiler would have to be run to determine what placeholder selectors are available within the context. My understanding of the underlying vscode-css-languageservice that this plugin uses, is that it is just a text parser looking for specific symbol structures. It may be possible only support directly declared placeholder selector names, but this doesn't cover the most useful use cases for placeholder selectors IMO. I'd love to hear additional feedback on this.

**Comment by KraXen72** [comment]
please add atleast autocomplete support for directly defined placeholder selectors when using @extend thanks

**Comment by stuffaboutpete** [comment]
Agreed that this would be a very powerful feature - the directly defined version is plenty.

Is this project stale?

**Comment by yorkshire-pudding** [comment]
Yes, please.
No idea why this can't be done as for directly defined ones they follow a standard pattern.  Perhaps, even if we had to set the file and folder to look in, that would be better than nothing.  I put most of my placeholders in `abstracts/_placeholders.scss` and  these are then used in most file (along with other abstracts via `_index.scss`); the ones that are in other files are generally used just in that file so intellisense works on these anyway.
